test/hdfs_test.py

```diff
@@ -32,6 +33,43 @@ class TestException(Exception):
 
 
 @attr('minicluster')
+class ConfigurationTest(MiniClusterTestCase):
+
+    def tezt_rename_dont_move(self, client):
```

Comment:
Hmm, it's so much like a `test_`, but it's not as I want to run it once for each configuration. Hmm..